android/data/config/firebase/google-services.test.json

```diff
@@ -60,10 +76,10 @@
               "client_type": 3
             },
             {
-              "client_id": "148482003030-485q62hqnch3m2mnue1ps0jdl35ikg7a.apps.googleusercontent.com",
+              "client_id": "148482003030-7anom7nbfkqlmk9oicnsmds7lsq3hbvc.apps.googleusercontent.com",
               "client_type": 2,
               "ios_info": {
-                "bundle_id": "app.raha.mobile"
+                "bundle_id": "app.raha.mobile.ios"
```

Comment:
Not sure why this one changed :/

Reply:
that changed because `app.raha.mobile` for some reason was not ok with apple so i did that to make it unique when we first published